workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -0,0 +1,20 @@
+from django.utils import timezone
+from data_refinery_models.models import Batch, BatchStatuses, DownloaderJob
+
+
+def start_job(job: DownloaderJob):
+    """Record in the database that this job is being started. """
+    job.worker_id = "For now there's only one. For now..."
```

Comment:
TBH I haven't thought this all the way through yet. Celery does assign a worker id to each worker however I would think that every time it is restarted these might be lost. There may be an instance id we can get through amazon which might help out. I mention [here](https://github.com/data-refinery/data_refinery/blob/master/data_models/data_refinery_models/models.py#L120) that this part of the model is still up in there air. I have only already included this to both remind myself to change it at some point and to make these functions seem necessary.